core/mondoo-azure-security.mql.yaml

```diff
@@ -52,7 +63,297 @@ policies:
+           * Cloud apps or actions --> All cloud apps 
+        5. Make sure that the policy is Enabled. You can also set Enable policy to Report-only, and After confirming your settings using report-only mode, an administrator can move the Enable policy toggle from Report-only to On. 
+    query: |
+      microsoft.security.latestSecureScores.controlScores.one( _['controlName'] == 'AdminMFAV2' && _['score'] == 10 )
+  - uid: mondoo-azure-security-ensure-that-between-two-and-four-global-admins-are-designated
+    title: Ensure that at least three global admins are designated and not more than four
+    severity: 60
+    docs:
+      desc: |
+        This control checks that there are enough Global Admins in a single tenant. 
+        When it comes to designating global admins, it's important to consider the size and complexity of the organization, as well as the level of responsibility and authority required for the role. As a general rule, it's a good idea to have at least three global admins to ensure that there is redundancy and coverage in case one admin is unavailable or leaves the organization.
+
+        At the same time, having too many global admins can lead to confusion and inefficiency, as multiple people may be making decisions or taking actions without proper coordination. Therefore, it's recommended to keep the number of global admins to no more than four, unless the organization is particularly large or complex and requires more administrators to properly manage its operations.
+      audit: |
+        __cnquery run__
+
+        To audit Microsoft Azure with `cnquery run`:
+
+        Run the following query:
+
+          ```bash
+          cnquery run azure -c "microsoft.rolemanagement.roleDefinitions.where(displayName == "Global Administrator").all(assignments.length > 1 && assignments.length <= 4)" --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id>
+          ```
+        __cnquery shell__
+
+        To audit Microsoft azure with `cnquery shell`:
+
+        1. Launch `cnquery shell`:
+
+          ```bash
+          cnquery shell azure --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id>
+          ```
+        
+        2. Run the following query:
+
+          ```mql
+          microsoft.rolemanagement.roleDefinitions.where(displayName == "Global Administrator").all(assignments.length > 1 && assignments.length <= 4)
+          ```
+      remediation: |
+        ### Microsoft Azure Portal
+
+        To update via the Microsoft Azure portal:
+         1. Log in as a Global Administrator to the Microsoft Azure portal at https://portal.azure.com 
+         2. browse to Azure Active Directory
+         3. Users --> New user  
+         4. Create a new Global Admin, if there are not enough by: 
+            * Select the User's name
+            * roles --> Global Administrator -> Save changes
+         5. Remove the Global Admin, if there are more than four Global Administrators
```

Comment:
```suggestion
         5. If there aren't enough global admins, add one:
```

Reply:
Thanks, This is for removing Extra Global Admins!